bokeh/util/compiler.py

```diff
@@ -278,8 +241,6 @@ def implementation(self):
         if isinstance(impl, str):
             if "\n" not in impl and impl.endswith(exts):
                 impl = FromFile(impl if isabs(impl) else join(self.path, impl))
-            else:
-                impl = CoffeeScript(impl)
```

Comment:
Should this default to `TypeScript` now?

Reply:
Yes.